### ISSUE TITLE
Expose apps' accessed data types

### ISSUE THREAD

**Issue by kasnder** [emanuele-f/PCAPdroid#240]
Hello,

PCAPDroid is a great tool. Especially with the new TLS decryption functionality it promises to help the investigation of apps' data practices a lot.

One feature that seems to be missing (and I've been talking to quite a few people about this) is an ability to get insights into what types of data get shared by apps.

This feature would be enormously helpful for researchers, journalists and NGOs seeking to investigate apps' data practices.

**Comment by emanuele-f** [comment]
Hi, do you mean something like domain classification to outline, for example, analytics domains?

**Comment by kasnder** [comment]
I mean something like Android Advertising ID, IMEI, Android ID, current location, phone numbers.

**Comment by emanuele-f** [comment]
Yeah, it would be useful to show this info. Do you already have some ideas on how to implement this? Does it require additional permissions for PCAPdroid?
